### PR TITLE
Enhancement: Include `*.inc` files (with exceptions)

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] includes `*.inc` files when running `friendsofphp/php-cs-fixer`
- [x] runs `make coding-standards`

Follows #559.

💁‍♂️ A lot of whitespace changes, so perhaps best reviewed using https://github.com/php/web-php/pull/662/files?w=1.

### CODE REVIEW

File: include/pregen-confs.inc [hunk 1]
```diff
@@ -1,13 +1,5 @@
...
-
-
-
-
-
-
-
-
 /*
  * DO NOT MODIFY THIS FILE
```
Comment (localheinz) [inline include/pregen-confs.inc:4]: Is it?

**Review by kamil-tekiela — APPROVED** [review]
I think it makes sense. I can see only whitespace changes.

**Comment by cmb69** [comment]
Thank you!

**Comment by localheinz** [comment]
Thank you, @cmb69 and @kamil-tekiela!

**Comment by cmb69** [comment]
Oh, that failed the CS check: https://github.com/php/web-php/runs/7357443033?check_suite_focus=true. Should be fixed by https://github.com/php/web-php/commit/5c9eab2f382599db143fb491f8ee7762960e8ab3.

**Pull request merged**